src/__tests__/timerUtils.js

```diff
@@ -0,0 +1,27 @@
+import { setTimeout } from '../helpers/getTimerFuncs';
+
+const FakeTimerTypes = [
+  'default',
+  'legacy',
+  // 'modern', // broken for now
+];
```

Comment:
Would it make more sens to expose it as an enum-like object: `{ Default 'default', Legacy: 'legacy', ...}`

Reply:
Sounds good, I've made the changes